northbound/sensorthings/mqtt/integration-test.bndrun

```diff
@@ -70,4 +71,5 @@
 	org.osgi.util.promise;version='[1.3.0,1.3.1)',\
 	org.osgi.util.pushstream;version='[1.0.2,1.0.3)',\
 	slf4j.api;version='[1.7.36,1.7.37)',\
-	slf4j.simple;version='[1.7.36,1.7.37)'
\ No newline at end of file
+	slf4j.simple;version='[1.7.36,1.7.37)'
+-runblacklist: bnd.identity;id='slf4j.reload4k'
```

Comment:
This looks like a merge issue. We shouldn't have two blacklists

Reply:
I've missed that as I added my runbacklist. Done